fontforge/autosave.c

```diff
@@ -112,7 +112,7 @@ return( false );
 	    fprintf( stderr, " Done\n" );
 	}
     }
-    free(sf);
+    SplineFontFree(sf);
```

Comment:
I guess you'll need to include `splineutil.h` for this. Quote-style after `baseviews.h` should probably work.

Reply:
As usual, thank you for your feedback.
I'll patch this when I get back home after my daytime job.

Reply:
Forgot to tell I pushed a patch on my local fork about this point.
It should be ok now.